DigiMeSDK/Repository/Classes/Objects/Social/Post.swift

```diff
@@ -56,7 +56,9 @@ public class Post: NSObject, BaseObjectDecodable {
     public let likeCount: Int
     public let links: [Link]?
     public let longitude: Double
-    public let originalPostIdentifier: String?
+    public var originalPostIdentifier: String? {
+        return String(describing: originalPostIdentifierRaw)
```

Comment:
This (and the other functions using `String(describing:)`) should check whether the raw value is nil first, otherwise the string returned is `"nil"`!

i.e.
```
guard let x = x else {
    return nil
}

return String(describing: x)
```